dockertest/docker_daemon_unittests.py

```diff
@@ -66,5 +66,56 @@ def value_to_json(value):
         self.assertEqual(i.get_json('bar'), [{u'foo': u'bar'}])
         self.assertEqual(i.interface, None)
 
+
+class TestStringEdit(unittest.TestCase):
+    pass
+
+    def test_bad_input_no_prefix(self):
+        import docker_daemon
+        self.assertRaises(ValueError,
+                          docker_daemon.edit_options_string, "OPTINOS=hi")
+
+    def test_bad_input_mismatched_quotes(self):
+        import docker_daemon
+        self.assertRaises(ValueError,
+                          docker_daemon.edit_options_string,
+                          "OPTIONS='missing end quote")
```

Comment:
Hmmmm, yep, totally running away from mock_open() aren't ya :grinning: 

I don't blame you.